### PR TITLE
Fix set_state implementation for command argument instance.

### PR DESCRIPTION
**Pull request by prmukherj**

### CODE REVIEW

File: src/ansys/fluent/core/services/datamodel_se.py [hunk 1]
```diff
@@ -1132,6 +1132,17 @@ def get_state(self) -> Any:
 
     getState = get_state
 
+    def set_state(self, state) -> Any:
+        """Get state of the command argument."""
```
[the remark this answers is not in the document]
Comment (prmukherj) [reply src/ansys/fluent/core/services/datamodel_se.py]: Thank you @raph-luc, good catch.

File: src/ansys/fluent/core/services/datamodel_se.py [hunk 2]
```diff
@@ -1132,6 +1132,17 @@ def get_state(self) -> Any:
...
+        parent_state = self.parent.get_state()
+        try:
+            parent_state[self.name] = state
+            self.parent.set_state(parent_state)
+        except KeyError:
+            pass
```
Comment (raph-luc) [inline src/ansys/fluent/core/services/datamodel_se.py:1142]: Question unrelated to this PR: why are we ignoring `KeyError` exceptions here? Should we at least log these?

Comment (prmukherj) [reply src/ansys/fluent/core/services/datamodel_se.py:1142]: I think we should.

cc. @mkundu1, @seanpearsonuk

File: src/ansys/fluent/core/services/datamodel_se.py [hunk 3]
```diff
@@ -1132,6 +1132,17 @@ def get_state(self) -> Any:
 
     getState = get_state
 
+    def set_state(self, state) -> Any:
+        """Set state of the command argument."""
+        parent_state = self.parent.get_state()
+        try:
+            parent_state[self.name] = state
+            self.parent.set_state(parent_state)
+        except KeyError:
```
Comment (mkundu1) [inline src/ansys/fluent/core/services/datamodel_se.py:1141]: In which scenario will we get a KeyError?

If the KeyError happens due to some user error, we should throw the appropriate exception. If it never happens (guaranteed by the implementation), we should remove the try-except.

File: src/ansys/fluent/core/services/datamodel_se.py (hunk 2 above)
Comment (prmukherj) [reply src/ansys/fluent/core/services/datamodel_se.py:1142]: Done.

**Review by seanpearsonuk — APPROVED** [review]

**Review by hpohekar — APPROVED** [review]

**Review by raph-luc — APPROVED** [review]

**Review by mkundu1 — APPROVED** [review]

**Pull request merged**